tools/Raven.Debug/CommandLineApp.cs

```diff
@@ -76,9 +93,11 @@ public static int Run(string[] args)
                     if (outputOption.HasValue())
                         output = outputOption.Value();
 
+                    var includeStackObjects = includeStackObjectsOption.Values.FirstOrDefault() == "on";
```

Comment:
It's case sensitive, on purpose?

Reply:
this is the result of when you pass a command line option with no value:
Raven.Migrator --includeStackObjects